src/library_html5_webgl.js

```diff
@@ -80,24 +80,23 @@ var LibraryHtml5WebGL = {
 #if ASSERTIONS
     assert(attributes);
 #endif
-    var a = attributes >> 2;
-    var powerPreference = HEAP32[a + ({{{ C_STRUCTS.EmscriptenWebGLContextAttributes.powerPreference }}}>>2)];
```

Comment:
This set of changes look like just changing one form of i32 memory reference to another; where's the 64-bit fix?

Reply:
The problem is that >>2 doesn't work for numbers bigger and 2**32.